pages/mesosphere/dcos/2.1/installing/evaluation/index.md

```diff
@@ -4,7 +4,7 @@ navigationTitle:  Cloud Installation
 title: Cloud Installation
 menuWeight: 10
 excerpt: Guide to Installing DC/OS on cloud environments using the Mesosphere Universal Installer
-model: /mesosphere/dcos/2.1/data.yml
+model: /mesosphere/dcos/2.2/data.yml
```

Comment:
Is this change for 2.2? If so, we need to move all these changes to the 2.2 branch, which has already been created on staging. Otherwise, if this change is for 2.1, we should remove this model name change.

Reply:
i think its for both... just needed to start somewhere